deepquest/train.py

```diff
@@ -99,6 +99,10 @@ def train_model(params, weights_dict, load_dataset=None, trainable_pred=True, tr
     #params['OUTPUT_VOCABULARY_SIZE'] = dataset.vocabulary_len[params['OUTPUTS_IDS_DATASET_FULL'][0]]
     params['OUTPUT_VOCABULARY_SIZE'] = dataset.vocabulary_len['target_text']
 
+    if params['RELOAD_EPOCH'] == True and params['RELOAD'] > 0:
+        compare_params(params, pkl2dict(os.path.join(params['STORE_PATH'], 'config.pkl')), ignore=['RELOAD', 'RELOAD_EPOCH'])
```

Comment:
No, this works so that if there is already a model with the same name, then instead of overwriting, we reload and resume training the model. This was handled here https://github.com/RSE-Sheffield/nmt-keras/pull/22

Reply:
(Have deleted a previous comment as I misunderstood something)

Scenario 1 is handled here.

Scenario 2 is not something I've considered yet.

Reply:
As it stands, this needs to be here because of where the params dictionary is written out and loaded. Currently all the params handling is otherwise done before train.main() is called.

Reply:
Have tested scenario 2 as of 90eb879 and it works!

Reply:
I don't have time to test it now, so I'll trust you on this one as you say it works.
Resolving.